### PR TITLE
Introduce delta xds log scope to distinguish ads logs

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
**Please provide a description of this PR:**

### CODE REVIEW

File: pilot/pkg/networking/core/v1alpha3/cluster.go [hunk 1]
```diff
@@ -173,7 +174,8 @@ func (configgen *ConfigGeneratorImpl) buildClusters(proxy *model.Proxy, req *mod
 	return resources, model.XdsLogDetails{AdditionalInfo: fmt.Sprintf("cached:%v/%v", cacheStats.hits, cacheStats.hits+cacheStats.miss)}
 }
 
-func shouldUseDelta(updates *model.PushRequest) bool {
```
Comment (ramaraochavali) [inline pilot/pkg/networking/core/v1alpha3/cluster.go:176]: This is supposed to be generic and used for all xDS?

Comment (hzxuzhonghu) [reply pilot/pkg/networking/core/v1alpha3/cluster.go:176]: no, it is only for cds

Comment (hzxuzhonghu) [reply pilot/pkg/networking/core/v1alpha3/cluster.go:176]: Otherwise it should not be in this file

Comment (ramaraochavali) [reply pilot/pkg/networking/core/v1alpha3/cluster.go:176]: I think I only added this and my intention was to reuse this across all xDS

**Review by howardjohn — COMMENTED** [review]
I can see why you sent the PR but I actually have a very slight preference to keep things the way they are. My thinking is that you will likely be all delta or all sotw, so we don't really need to distinguish then much. On initial connection it shows if it's delta or not so if you really need to you can trace it back.

That being said it's only a slight preference and I see benefits here as well so not opposed to merging

**Comment by hzxuzhonghu** [comment]
Yes， we can infer from the context, even some are full xds, some are delta. but not obvious for users.

File: pilot/pkg/networking/core/v1alpha3/cluster.go (hunk 1 above)
Comment (hzxuzhonghu) [reply pilot/pkg/networking/core/v1alpha3/cluster.go:176]: I will revert this  change, do it when necessary

**Review by ramaraochavali — APPROVED** [review]

**Pull request merged**